ports/mdl/vcpkg.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "mdl",
+  "version": "2021.1.2",
+  "description": "NVIDIA Material Definition Language SDK",
+  "homepage": "https://github.com/NVIDIA/MDL-SDK",
+  "license": "BSD-3-Clause",
+  "supports": "!arm & !x86 & !(windows & static)",
```

```suggestion
  "supports": "!arm & !x86 & !(windows & staticcrt)",
```

Does this need to be static or would staticcrt work?

---

Unfortunately it does not work either: current MDL-SDK upstream expects the `freeimage` dependency provided as a shared library, the issue is not solely about library linkage vs crt linkage.
I plan to address this limitation as a follow-up of this PR as the change set is already quite large.